app/factories/feedback.py

```diff
@@ -11,9 +11,9 @@ class Meta:
         model = Feedback
         sqlalchemy_session = db.session
 
-    event = factory.RelatedFactory(EventFactoryBasic)
+    session = factory.RelatedFactory(SessionFactory)
     user = factory.RelatedFactory(UserFactory)
     rating = "4"
-    comment = "Awesome event."
-    event_id = 1
+    comment = "Awesome session."
+    session_id = 1
     user_id = 2
```

Comment:
user_id 1 is the currently authenticated user. How can user 1 post session for 2? Checks aren't working correctly it seems

Reply:
Yeah. I somehow missed to set it to 1
But the test passed. That's weird
I'll make it to 1 anyway

Reply:
@iamareebjamal I've updated it to 1. User with ID `1` can change sessions for `2` because the checks implemented here gives access to someone with organizer access of an event, to create/update feedback for that event -
```
if session and not has_access('is_organizer', event_id=session.event_id):
                raise ForbiddenException({'source': ''},
                                         "Event organizer access required")
```